config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  get "/users" , to: "users#index",  as: "users"
+  get "/users/:id" , to: "users#show" , as: 'user'
+  get "/users/:id/posts" , to: "posts#index", as: "user_posts"
+  get "/users/:id/posts/:id" , to: "posts#show", as: "user_post"
```

Comment:
- [ ] [optional] I would suggest replacing the routes with the below code. It will be a clearer way of defining routes.
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end